src/hooks/useGlobalAppState.ts

```diff
@@ -142,6 +77,29 @@ const onStateChange = (
  */
 export const useGlobalAppState = (): void => {
   const client = useClient()
+  const initFlag = useRef(false)
+  // eslint-disable-next-line @typescript-eslint/no-unnecessary-condition
+  const routeIndex = useNavigationState(state => state?.index)
+
+  // On app start
+  useEffect(() => {
+    log.info('useGlobalAppState: app start')
+
+    if (getIsSecurityFlowPassed() && !initFlag.current) {
```

Comment:
securityFlow on app launch (not wakeup) is launched from homeView. Not great but looked a bit tricky to change now